docs/application/native/guides/ui/efl/mobile/ui-components.md

```diff
@@ -0,0 +1,55 @@
+# Mobile UI Components
+
+EFL Elementary is a set of fast, finger-friendly, scalable, and themeable UI component libraries. The number of supported UI components is around 80, including both [containers](./ui-layouts.md) and non-containers (listed in this topic). Originally, Elementary was developed as part of the Window Manager development on desktop devices. For the mobile profile, Tizen reused the proper UI components and created new ones, and then enhanced and adjusted all of them for Tizen native applications.
```

Comment:
Broken link. `./ui-layouts.md` should be changed into `../ui-layouts.md`.

Reply:
done